mm/Kconfig

```diff
@@ -69,6 +69,14 @@ config MM_DEFAULT_ALIGNMENT
 		memory default alignment is equal to sizoef(uintptr), if this value
 		is not 0, this value must be 2^n and at least sizeof(uintptr).
 
+config MM_NODE_PENDING
+	bool "Enable pending memory node"
+	default n
```

Comment:
depends on KASAN? only the additional guard bytes is only needed when kasan is enabled